docs/about.md

```diff
@@ -1,138 +1,137 @@
+<h1 class="display-3">About us</h1>
 
-We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg/).
+RepoSense is a project based in the [National University of Singapore, School of Computing](http://www.comp.nus.edu.sg/), and is funded by a _Teaching Enhancement Grant_ from [NUS Center for Development of Teaching and Learning](http://www.cdtl.nus.edu.sg/).
 
-## Current Team
+<!-- ==================================================================================================== -->
+
+## Current team
 
 ### [Damith C. Rajapakse](http://www.comp.nus.edu.sg/~damithch)
 ![](https://avatars.githubusercontent.com/u/1673303?s=150&v=4)<br/>
 **Role**: Project Advisor
 
----
+<!-- ------------------------------------------------------------------------------------------------------ -->
 
 ### [Peh Xian Bin, Eugene](https://github.com/eugenepeh)
 ![](https://avatars.githubusercontent.com/u/19277206?s=150&v=4)<br/>
 **Role**: Project Lead [2018 May - ]<br/>
 
----
+<!-- ------------------------------------------------------------------------------------------------------ -->
 
 ### [Teng Yong Hao](https://github.com/yong24s)
 ![](https://avatars2.githubusercontent.com/u/2003406?s=150&v=4)<br/>
 **Role**: Mentor [2018 Dec - ]<br/>
 Committer [2018 Aug - 2018 Dec]<br/>
 Contributor [2018 May - 2018 Aug]<br/>
 
----
+<!-- ------------------------------------------------------------------------------------------------------ -->
 
 ### [Wang Chao](https://github.com/fzdy1914)
 ![](https://avatars3.githubusercontent.com/u/35621726?s=150&v=4)<br/>
 **Role**: Committer [2019 May - ]<br/>
 Contributor [2018 Dec - 2019 May]<br/>
 
----
+<!-- ------------------------------------------------------------------------------------------------------ -->
 
 ### [James Pang Mun Wai](https://github.com/jamessspanggg)
 ![](https://avatars1.githubusercontent.com/u/32864116?s=150&v=4)<br/>
 **Role**: Contributor [2019 May - ]<br/>
 
----
+<!-- ------------------------------------------------------------------------------------------------------ -->
 
 ### [Lee Jin Yao](https://github.com/jylee-git)
 ![](https://avatars3.githubusercontent.com/u/35756209?s=150&v=4)<br/>
 **Role**: Contributor [2019 May - ]<br/>
 
----
+<!-- ------------------------------------------------------------------------------------------------------ -->
 
 ### [Liu Yiwen](https://github.com/0blivious)
 **Role**: Contributor [2019 August - ]<br/>
 
----
+<!-- ------------------------------------------------------------------------------------------------------ -->
 
 ### [Tejas Bhuwania](https://github.com/Tejas2805)
 ![](https://avatars2.githubusercontent.com/u/35946746?s=150&v=4)<br/>
 **Role**: Contributor [2019 August - ]<br/>
 
----
+<!-- ------------------------------------------------------------------------------------------------------ -->
 
 ### [Anubhav](https://github.com/anubh-v)
 **Role**: Contributor [2019 September - ]<br/>
```

Comment:
We can use @anubh-v 's profile pic if he wishes to do so:

```suggestion
### [Anubhav](https://github.com/anubh-v)
![](https://avatars1.githubusercontent.com/u/35621759?s=400&u=f1c240d85823576808943c93ea035166b5343a79&v=4)
**Role**: Contributor [2019 September - ]<br/>
```

Reply:
Thanks @jylee-git for your consideration and suggestion.
I would prefer to keep the current state.